BUILD.bazel

```diff
@@ -758,31 +758,56 @@ ray_cc_library(
 )
 
 ray_cc_library(
-    name = "stats_metric",
+    name = "stats_metric_core",
     srcs = [
         "src/ray/stats/metric.cc",
-        "src/ray/stats/metric_defs.cc",
-        "src/ray/stats/tag_defs.cc",
     ],
     hdrs = [
         "src/ray/stats/metric.h",
-        "src/ray/stats/metric_defs.h",
         "src/ray/stats/tag_defs.h",
     ],
     deps = [
         "//src/ray/util",
```

Comment:
I don't think we need this dependency, could you please try remove it?

Reply:
Missing dependency for: https://github.com/ray-project/ray/blob/e18a59ab6513f8332ce2abf1ed765c48ff2dce8a/src/ray/stats/metric.h#L25